packages/ui-form-field/src/FormField/index.tsx

```diff
@@ -68,7 +68,6 @@ class FormField extends Component<FormFieldProps> {
         label={this.props.label}
         vAlign={this.props.vAlign}
         as="label"
-        htmlFor={this.props.id}
```

Comment:
This is not needed because the `label`s child is the control, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#defining_an_implicit_label

Reply:
EDIT 2 weeks later: we should not do this, it will bring the last leaf on the focus causing issues with e.g. multiple select